### ISSUE TITLE
Outdated credentials listed under Data Types

### ISSUE THREAD

**Issue by bblazicevic** [bcgov/orgbook-bc-client#245]
We still see a couple of outdated credential items listed under Data Types, can we update this page to reflect active credentials only?

Ministry of Energy, Mines and Low-carbon Innovation - BC Mines Act Permit (we have a new credential in development)
Ministry of Environment and Climate Change Strategy - Annual GHG Emissions Report

Stephen's comments via email: "I'm not sure how the page is generated - probably based on what issuers are configured.  Perhaps the best fix, if that is the case, is to change the condition for being listed to "Configured AND Has more than 1 credential in OrgBook".  The latter condition would eliminate those two from showing"

**Comment by WadeBarnes** [comment]
Correct.  This list is constructed based on the registered issuers (and the credential types they've indicated they issue), whether or not any credentials have been issues by those issuers.  For example, apart from those you've identified, there is the BC Registries Address Credential.  Though BC Registries is an active issuer and has registered a schema for an Address Credential, no Address Credentials have ever been issued.

You can get details regarding the issuers and the credential types they've issued here; https://orgbook.gov.bc.ca/api/v2/

Examples:
- https://orgbook.gov.bc.ca/api/v2/issuer
- https://orgbook.gov.bc.ca/api/v2/issuer/1/credentialtype
- https://orgbook.gov.bc.ca/api/v2/schema
